sites/main-site/src/config/contributors.yaml

```
@@ -1094,3 +1094,16 @@ contributors:
     contact_github: sagc-bioinformatics
     location: [-34.921146, 138.589726]
     twitter: sa_genomics
+  - full_name: Memorial Sloan Kettering Cancer Center
+    short_name: MSKCC
+    description: >
+      The Sloan Kettering Institute (SKI) is the basic and translational research arm of Memorial Sloan Kettering Cancer Center. Discoveries made in SKI labs are an important driver of clinical progress at MSK and beyond.
+    address: 1275 York Ave, New York, USA. 10065
+    url: https://www.mskcc.org/research/ski
+    affiliation_url: https://www.mskcc.org
+    image_fn: mskcc.svg
+    contact: Juan E. Arango Ossa
+    contact_email: arangooj@mskcc.org
+    contact_github: juanesarango
+    location: [40.761454, -73.962245]
+    twitter: juanesarango
```

I think that this `twitter` is typically for the institution and not the contact person..

```suggestion
    twitter: MSKCancerCenter
```